### PR TITLE
Fix misc. issues with fluid appearances

### PR DESCRIPTION
**Pull request by out-of-phaze**
- Fixes an issue with fluids on recessed natural turfs flickering above/below mobs.
- Fixes torch head colors breaking when an alpha mask is applied.
- Fixes KEEP_TOGETHER never being unapplied when an alpha mask is removed, because remove_filter returned the wrong value.
- Adds held and worn icon update calls to torches when they're extinguished.
- Fixes ticks2readable giving a blank output for values under 10 ticks (relevant for ability cooldown etc).

### CODE REVIEW

File: code/game/objects/items/flame/_flame.dm [hunk 1]
```diff
@@ -125,6 +125,9 @@
 	update_attack_force()
 
 	update_icon()
+	if(ismob(loc)) // not very robust for things like accessories...
```
Comment (MistakeNot4892) [inline code/game/objects/items/flame/_flame.dm:128]: get_recursive_loc_of_type() or whatever that proc is called might be good here?

File: code/game/objects/items/flame/flame_torch.dm [hunk 2]
```diff
@@ -76,11 +76,11 @@
 	if(head_material)
 		var/decl/material/head_mat = GET_DECL(head_material)
 		if(burnt)
-			add_overlay(overlay_image(icon, "[icon_state]-burnt", head_mat.color, flags = RESET_COLOR))
+			add_overlay(overlay_image(icon, "[icon_state]-burnt", head_mat.color, flags = RESET_COLOR|KEEP_APART))
```
Comment (MistakeNot4892) [inline code/game/objects/items/flame/flame_torch.dm:79]: Does this work as expected? I had a lot of trouble with overlays like this on Polaris and I remember this not working.

**Review by MistakeNot4892 — APPROVED** [review]
Curious about the KEEP_APART comment but not really blocking.

File: code/game/objects/items/flame/flame_torch.dm (hunk 2 above)
Comment (out-of-phaze) [reply code/game/objects/items/flame/flame_torch.dm:79]: it worked fine in my testing

**Pull request merged**